proposals/exp-show-target.md

```diff
@@ -0,0 +1,210 @@
+- Enhancement Proposal PR: (leave this empty)
+- Contributors: dberenbaum, (add your Github handle)
+
+# Summary
```

Comment:
Meta: should the template have a proposal title?

Reply:
Minor since obvi the PR has one but when you see the rendered MD doc it's not so evident.

Reply:
Metameta: I can't resolve conversations in this repo @dberenbaum but feel free to collapse as you see fit.